### ISSUE TITLE
Consider modifgying advice for 1.4.13 Content on Hover and Focus

### ISSUE THREAD

**Issue by detlevhfischer** [appt-org/appt-org#3]
https://appt.org/en/guidelines/wcag/success-criterion-1-4-13

**Suggested improvements**
The text refers to content that pops up when elements are touched. This is not really what the SC covers, though - it covers mouse hover and keyboard focus receipt.

The solution of adding a close button would not work snce ot would mean users have to move focus to activate it.
The SC applies when apps are used with attached mice or keyboard (even though hover and focus will be rare in apps for obvious reasons).

**Comment by JJdeGroot** [comment]
Hi @detlevhfischer thanks for your suggestion. I'll discuss it with other project members in April, when I'm back from my holiday.

**Comment by JJdeGroot** [comment]
Hi @detlevhfischer , like you mentioned, pointer hover is something that doesn't really happen in apps. Keyboard focus is probably more common. Long press might be most equivalent to 'pointer hover' as intended in the SC. Perhaps we can rename touch/click to 'long press' and/or keyboard focus. What do you think?

**Comment by JJdeGroot** [comment]
Hi @detlevhfischer what do you think of the text suggested on April 11?

**Comment by detlevhfischer** [comment]
@JJdeGroot Hover is definitely an option when a USB or bluetooth mouse is attached to a tablet, even though hover events it will be rarely supported since most users do use touch to interact with the app. But mobile OS become more prevalent and contain more complex applications where the use of keyboard or mouse might make sense. So it is not clear that hover can simply be discounted.

As to your long press idea: There simply are as yet no particular requirements for long press contained in 1.4.13 - so if you include requirements for that, it should be clear that this is not a normative requirement from WCAG 2.X or even the current update of [WCAG2ICT](https://w3c.github.io/wcag2ict/#content-on-hover-or-focus), but something you consider worth including on its own merit.

**Comment by JJdeGroot** [comment]
Hi @detlevhfischer  I have changed the sentence to:

English:

Ensure it is possible to hide information which appears automatically after hovering with the pointer or focusing with assistive technologies.

Dutch:

Zorg dat het mogelijk is om informatie te verbergen indien deze automatisch verschijnt na hover met de aanwijzer of focus met hulpmiddelen.

This sentence is now more in line with the official WCAG definition.

**Comment by detlevhfischer** [comment]
@JJdeGroot I think apart from hovering (pointer) focusing should not be restricted to "focusing with assistive technologies". The main case for th ekeyboard focus half of 1.4.13 will be checking keyboard operation of an app by coupling it with an external keyboard, focusing elements with tab (or arrow keys as it may be), and checking that any content that pops up on kb focus can be closed (e.g via ESC). The focus of 1.4.13 is clearly on sighted users, not screenreader users. I agree that content appearing when focusing elements in read mode (e.g. via touch swipe gestures) should probably also meet the requirements, but since WCAG does not spell out if the same requirement applies to content when a screen reader is active, I would avoid any explicit reference to assisitve technology use here.

**Comment by JJdeGroot** [comment]
Hi @detlevhfischer good points, I have changed assistive technologies to keyboard. Also updated some more phrases where assistive technologies were mentioned.
